genesis-blocks/docker-entrypoint.sh

```diff
@@ -0,0 +1,16 @@
+#! /bin/bash
+set -o errexit
+set -o pipefail
+
+# wait until there's a optimist instance up
+while ! nc -z ${CLIENT_HOST:-client} ${CLIENT_PORT:-80}; do sleep 3; done
+while true; do 
```

Comment:
I might be missing something, why did we not use `nc` for this as below
`while ! nc -z ${DEPLOYER_HOST:-client} ${DEPLOYER_PORT:-80}; do sleep 3; done`